pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -903,6 +923,16 @@ type KubeAPIServerConfig struct {
 	// This field is only available for Kubernetes v1.30 or later.
 	// +optional
 	StructuredAuthorization *StructuredAuthorization `json:"structuredAuthorization,omitempty" protobuf:"bytes,18,opt,name=structuredAuthorization"`
+	// Autoscaling contains auto-scaling configuration options for the kube-apiserver.
+	// +optional
+	Autoscaling *ControlPlaneAutoscaling `json:"autoscaling,omitempty" protobuf:"bytes,19,opt,name=autoscaling"`
+}
+
+// ControlPlaneAutoscaling contains auto-scaling configuration options for control-plane components.
+type ControlPlaneAutoscaling struct {
+	// MinAllowed configures the minimum allowed resource requests.
+	// +optional
+	MinAllowed corev1.ResourceList `json:"minAllowed,omitempty" protobuf:"bytes,1,rep,name=minAllowed,casttype=ResourceList,castkey=ResourceName"`
```

Comment:
I deleted the protobuf tag and re-run `make generate WHAT=protobuf`.

It generated:
```suggestion
	MinAllowed corev1.ResourceList `json:"minAllowed,omitempty" protobuf:"bytes,1,rep,name=minAllowed,casttype=k8s.io/api/core/v1.ResourceList,castkey=k8s.io/api/core/v1.ResourceName"`
```


https://github.com/gardener/gardener/blob/ee88ac263a9bf7c36e5240de1ceddddb065be1c0/docs/development/changing-the-api.md#L26 suggests:
> Do not copy protobuf tags from other fields but create them with `make generate WHAT="protobuf"`.